src/vs/workbench/contrib/notebook/browser/services/notebookServiceImpl.ts

```diff
@@ -202,8 +222,37 @@ export class NotebookProviderInfoStore extends Disposable {
 					cellOptions = (options as INotebookEditorOptions | undefined)?.cellOptions;
 				}
 
-				const notebookOptions: INotebookEditorOptions = { ...options, cellOptions, viewState: undefined };
-				const editor = NotebookEditorInput.getOrCreate(this._instantiationService, notebookUri, preferredResource, notebookProviderInfo.id);
+				let notebookOptions: INotebookEditorOptions;
+
+				if (resource.scheme === Schemas.vscodeNotebookCellOutput) {
+					if (!data?.handle) {
+						throw new Error('Invalid cell handle');
+					}
+					const cellUri = data?.notebook.with({ fragment: resource.fragment });
```

Comment:
This cell Uri is not correct, as the scheme is not `vscodeNtoebookCell`
You will need to re-generate this using `CellUri.generate`